packages/webapp/src/components/Form/InputBase/InputBaseField/index.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { ComponentPropsWithoutRef, ReactElement, forwardRef } from 'react';
+import { type InputBaseProps } from '../';
+import styles from './styles.module.scss';
+import clsx from 'clsx';
+
+type InputBaseFieldProps = Pick<InputBaseProps, 'leftSection' | 'rightSection'> & {
+  crossIcon?: ReactElement;
```

Comment:
Would something like `resetIcon` be better? Because the input can have an initial value and in that case clicking this icon wouldn't clear the entire value but reset it to its initial state.